discovery/kubernetes/client_metrics.go

```diff
@@ -127,12 +128,12 @@ var (
 		},
 		[]string{"queue_name"},
 	)
-	clientGoWorkqueueLatencyMetricVec = prometheus.NewSummaryVec(
```

Comment:
Originally, we [decided](https://github.com/prometheus/prometheus/pull/4458#pullrequestreview-148651324) on quantile-less summaries to cut down on the number of time-series. But it might be worth adding a comment at the top of the `var()` block.

Reply:
Then I'll revert this change. It'll still work if it's declared as a Summary.